KISFrenchTranslationPatch

```diff
@@ -0,0 +1,7 @@
+{
+    "spec_version": "v1.4",
+    "x_via": "Automated SpaceDock CKAN submission",
+    "license": "All Rights Reserved",
```

Comment:
Will need to be `"restricted"` rather than "All Rights Reserved".
Also, this should depend on KIS, please add the following, making sure that all but the last stanza have commas at the end:

    "depends": [
        { "name": "KIS" }
    ]